apps/example-app/app/devtools/ReactotronConfig.ts

```diff
@@ -59,7 +61,7 @@ reactotron.onCustomCommand({
   command: "showDevMenu",
   handler: () => {
     Reactotron.log("Showing React Native dev menu")
-    NativeModules.DevMenu.show()
+    TurboModuleRegistry.getEnforcing<DevMenuSpec>("DevMenu")?.show()
```

Comment:
Typically the TM defining library should wrap this `get` call and export it for you. Looks like it should be available from here: https://github.com/facebook/react-native/blob/eb7c43965ed8324aa50e5eab4a6184fd73f217e6/packages/react-native/Libraries/NativeModules/specs/NativeDevMenu.js#L13, which is pointing to https://github.com/facebook/react-native/blob/eb7c43965ed8324aa50e5eab4a6184fd73f217e6/packages/react-native/src/private/specs/modules/NativeDevMenu.js#L13-L23.

Reply:
The one thing I'm confused by there is I don't see anything else importing that module? Like I'm not sure how `show` gets called, like when opening the menu, in the current codebase I was searching. May not be in use?

Reply:
https://github.com/facebook/react-native/blob/eb7c43965ed8324aa50e5eab4a6184fd73f217e6/packages/react-native/src/private/specs/modules/NativeDevMenu.js

Reply:
Yeah something seems off, the calls work though 🤷

Reply:
@frankcalise The import you used is the second link I included, but I think the first link is where you're meant to import it from, because the second one says `private` in the path.

Reply:
An issue has been opened related to this: https://github.com/facebook/react-native/issues/46679